<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор типо айфон RTX off</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color:&quot;white&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_minus">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	background-color: rgb(222, 120, 37);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_num_5">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="pushButton_result">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	background-color: rgb(222, 120, 37);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_mod">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   c
	color: rgb(0, 0, 0);
	
	background-color: rgb(124, 124, 124);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_num_4">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_num_3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_num_7">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_num_1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_num_6">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_num_0">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_dot">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_num_2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_reroll">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   c
	color: rgb(0, 0, 0);
	
	background-color: rgb(124, 124, 124);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_del">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	background-color: rgb(222, 120, 37);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>÷</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_num_8">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_num_9">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	
	
	background-color: rgb(62, 62, 62);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_plus">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	background-color: rgb(222, 120, 37);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_multiply">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   color:&quot;white&quot;;
	background-color: rgb(222, 120, 37);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_change_tip">
      <property name="styleSheet">
       <string notr="true">QPushButton {
   c
	color: rgb(0, 0, 0);
	
	background-color: rgb(124, 124, 124);
border-radius:5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
